src/SDKs/CognitiveServices/dataPlane/SpellCheck/BingSpellCheck/Microsoft.Azure.CognitiveServices.SpellCheck.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.SpellCheck</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services SpellCheck API.</Description>
+    <VersionPrefix>1.1.0-preview</VersionPrefix>
```

Comment:
@ashku-ms if this is preview, it should start from 0.18.0 anything less than 1.0